projects/rero/ng-core/src/lib/record/record-ui.service.ts

```diff
@@ -172,4 +175,32 @@ export class RecordUiService {
 
     return of({ can: true, message: '' });
   }
+
+  /**
+   * Redirects after saving a resource
+   * @param pid - string, pid of the record
+   * @param record - object, record to save
+   * @param recordType - string, Type of resource
+   * @param action - string, http action: create or update
+   */
+  redirectAfterSave(pid: string, record: any, recordType: string, action: string) {
+    const config = this.getResourceConfig(recordType);
+    if (config.redirectUrl) {
+      config.redirectUrl(record).subscribe((result: string) => {
+        if (result !== null) {
+          this.router.navigate([result]);
+        }
+      });
+    } else {
+      // Default behaviour
+      if (action === 'update') {
+        // Relative path not working
+        // this.router.navigate(['../../detail', record.pid], {relativeTo: this.route, replaceUrl: true});
+        this.router.navigate(['records', recordType, 'detail', pid], {replaceUrl: true});
```

Comment:
We can't to this, because route ressource can have a link prefix, that we don't know.

Reply:
Done